target/linux/ramips/image/mt7621.mk

```diff
@@ -973,6 +973,16 @@ define Device/gnubee_gb-pc2
 endef
 TARGET_DEVICES += gnubee_gb-pc2
 
+define Device/hanyang_hyc-g920
+  $(Device/dsa-migration)
```

Comment:
New device. Migration is not needed since there is nothing to migrate from

Reply:
I think it is needed.

If we set it this will be added:
````
DEVICE_COMPAT_VERSION := 1.1
````
Without this the version will be set to 1.0 in `include/image.mk`

When the system booted up the compat version will be set to 1.1, see here:
target/linux/ramips/mt7621/base-files/etc/board.d/05_compat-version

Reply:
@hauke well yep with that file we are stuck in adding a compat everywhere... stupid thing and should have done correctly but you are right!

@momothefox Ignore this comment!